107/src/main/java/org/ehcache/jsr107/internal/tck/Eh107MBeanServerBuilder.java

```diff
@@ -42,7 +40,7 @@ public Eh107MBeanServerBuilder() {
   @Override
   public MBeanServer newMBeanServer(String defaultDomain, MBeanServer outer, MBeanServerDelegate delegate) {
     MBeanServerDelegate decoratingDelegate = new Eh107MBeanServerDelegate(delegate);
-    return JmxMBeanServer.newMBeanServer(defaultDomain, outer, decoratingDelegate, false);
```

Comment:
Why not `return super.newMBeanServer(defaultDomain, outer, decoratingDelegate)` since it extends MBeanServerBuilder already?